stories/button/linkButton/index.tsx

```diff
@@ -9,13 +9,20 @@ import { COLORS } from 'src/constants';
 import Container from '../container';
 
 // Mock React-Router Link
-/* eslint-disable */
-const MockLink = ({ to, children, ...rest }) => (
+const MockLink = ({
+  to,
+  children,
+  ...rest
+}: {
+  to: string;
+  children: React.ReactNode;
+  rest: any[];
```

Comment:
Since this is a rest operator within named parameters the shape is the remaining props within a single object, rather than an array, so let's re-type this to `{ [key: string]: unknown }`.